### ISSUE TITLE
bug: App shows `waitForResume` in the run log instead of human-friendly text if pause has no message

### ISSUE THREAD

**Issue by SyntaxColoring** [Opentrons/opentrons#10948]
# Steps to reproduce

Upload this protocol to a robot and view the run log.

```python
metadata = {"apiLevel": "2.12"}

def run(protocol):
    pipette = protocol.load_instrument("p300_multi_gen2", mount="left")
    lw1 = protocol.load_labware("opentrons_96_tiprack_300ul", 1)
    lw2 = protocol.load_labware("opentrons_96_tiprack_300ul", 3)
    for i in range(10):
        protocol.pause()
        for i in range(2):
            pipette.move_to(lw1["A1"].top())
            pipette.move_to(lw2["A12"].top())
```

Tested on `edge` 02c7c79cf908616f8831bad77f7af461a5852edf.

# Current behavior

The `protocol.pause()` steps show up as `waitForResume` in the run log.

<img width="1024" alt="Screen Shot 2022-06-29 at 12 12 58 PM" src="https://user-images.githubusercontent.com/3236864/176487449-3fdd6d7e-1c19-4986-b929-4550b4b02f89.png">

# Expected behavior

Those steps should show up as something human-friendly, like "paused", with the optional pause message.

**Comment by SyntaxColoring** [comment]
PR #10852 deprecated `pause` and replaced it with `waitForResume`. Maybe something in the app is still looking for a hard-coded `pause` string?

**Comment by mcous** [comment]
Investigated because I made sure to update #10852 to check for both `pause` and `waitForResume`. Turns out this is due to this logic:

https://github.com/Opentrons/opentrons/blob/1716862e2ee5d29813c520241322c620df85b629/app/src/organisms/Devices/ProtocolRun/StepText.tsx#L123-L127

If the command has no message in the payload (the message is optional), the run log uses the `commandType` as the default message. We just didn't notice before because the command type used to be `pause`.

There should probably be a default "Waiting for resume" i18n string available instead of defaulting to the command type
